brightcove/brightcove-popup/src/components/gridLayout.tsx

```diff
@@ -62,16 +63,25 @@ const GridLayout: React.FC<Layout> = (props) => {
                       </label>
                     </div>
                     <div className='item'>
-                      <span
-                        className='img'
-                        style={{
-                          backgroundImage: `url(${
-                            video.images.thumbnail.src
-                              ? video.images.thumbnail.src
-                              : video.images.thumbnail.source[0].src
-                          })`,
-                        }}
-                      ></span>
+                      {video.images.thumbnail ? (
+                        <span
+                          className='img'
+                          style={{
+                            backgroundImage: `url(${
+                              video.images.thumbnail.src
+                                ? video.images.thumbnail.src
```

Comment:
What if `video.images.thumbnail.src` or `video.images.thumbnail.source[0].src` is undefined or null ?
Please handle the falsy values here.
You can do it by using `lodash/get` library.
https://lodash.com/docs/4.17.15#get